project/src/main/kotlin/org/ionproject/core/classSection/ClassSectionController.kt

```diff
@@ -0,0 +1,25 @@
+package org.ionproject.core.classSection
+
+import org.ionproject.core.common.SIREN_MEDIA_TYPE
+import org.ionproject.core.common.Siren
+import org.ionproject.core.common.Uri
+import org.ionproject.core.klass.KlassController
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+class ClassSectionController(private val repo: ClassSectionRepo) {
+
+    @ExceptionHandler
+    fun handleClassNotInDbException(exception: ClassSectionNotInDbException) =
+        ResponseEntity.notFound().build<Any>()
```

Comment:
Let the global exception "ResourceNotFound" handle this type of exceptions instead of creating one per controller.